src/Client/app/(admin)/exams/review/page.tsx

```
@@ -424,21 +477,17 @@ const updateProblemSubmission = (
-                                                        className="w-16"
+                                                        className="w-12"
                                                         value={submission.score}
                                                         onChange={(e) =>
                                                             updateWrittenSubmission(
                                                                 submission.questionId,
-                                                                {score: parseInt(e.target.value)}
-                                                            )
-                                                        }
-                                                    />
-                                                    /{questionsData[submission.questionId].score}
+                                                                {score: parseInt(e.target.value)})
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)